scripts/check_compliance.py

```diff
@@ -675,6 +701,9 @@ def parse_args():
     parser.add_argument('-l', '--list', action="store_true",
                         help="List all test modules.")
 
+    parser.add_argument('-n', '--dryrun', action="store_true",
+                        help="Dryrun, do not post anything to github.")
```

Comment:
Hmm... looking at this PR fresh, I'm confused here.

Running `check_compliance.py` by itself never posts to GitHub, so what does option do differently?

I'll figure it out, but might be nice to have a longer help text to clarify it.

Reply:
It would be nice to have documentation for how the whole Shippable thing fits into things somewhere too, but that's unrelated.

Reply:
with all the changes coming in, I want to use this to see how the message we post to github looks like, maybe dryrun is the wrong option here, it is intended to be used with --github option and "pretend" we are posting to github. This is not not an "end-user" feature, this is for us to see how things end up on github.

Reply:
Maybe call it --github-dry-run?

Reply:
Could mention that it's meaningful together with `--github` too. "If used together with --github, blah blah".